### DISCUSSION TITLE
Get ruleset from league

### DISCUSSION THREAD

**Discussion by ThomasAdriaanse** [cwendt94/espn-api#325] [ESPN API]
Is there any way to obtain the ruleset for a specific league? I am  If not do you plan on implementing this?
Thanks a lot for making this, it is really easy to use for a python noob like me!

**Comment by cwendt94** [comment]
What type of rules are you thinking, like all of the league settings? If so I don't think it would be hard to add more.

There is already a settings object, these are the fields https://github.com/cwendt94/espn-api/wiki/Settings-Class

**Reply by ThomasAdriaanse** [reply]
I mean roster size, which (and how many) positions available on each roster, fantasy points per stat category, pretty much the data on the settings page.

**Reply by cwendt94** [reply]
Got it, I should definitely be able to expand on the settings class and add those fields.
